bin/public/installLogrotate.py

```diff
@@ -44,4 +45,6 @@ def install_logrotate(args):
     sc = scOpen("/etc/logrotate.conf")
     sc.replace('#compress', 'compress')
 
+    x("cp --remove-destination {0}syco-private/var/logrotate/* /etc/logrotate.d/".format(constant.SYCO_USR_PATH))
```

Comment:
This would break if the private syco plugin is called anything except syco-private. Please add support for any plugin directory name containing a /var/logrotate folder...

If you want inspiration, installSyco.py has something similar starting on row 80....